arcade/gui/widgets/layout.py

```diff
@@ -22,6 +22,14 @@ class UIAnchorLayout(UILayout):
     - anchor_y: str = None - uses `self.default_anchor_y` as default
     - align_y: float = 0
 
+    Usage:
+
+    .. code::py
+        manager: UIManager
+        anchor = manager.add(UIAnchorLayout())
+
+        anchor.add(child=child, ...)
+
```

Comment:
Usually this is enough
```
    Usage::

        manager = UIManager()
        anchor = manager.add(UIAnchorLayout())
        anchor.add(child=child, ...)
```

Reply:
Updated in https://github.com/pythonarcade/arcade/pull/1616/commits/edbb9a13e0d70ca7045432198d3330a7501ffeb2